streaming/src/main/scala/org/apache/gearpump/streaming/task/TaskUtil.scala

```diff
@@ -53,6 +53,10 @@ object TaskUtil {
     else t1
   }
 
+  def min(t1: Instant, t2: Instant, t3: Instant): Instant = {
```

Comment:
If this is a util, then how about making it more general by accepting *varargs* ?